forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -7061,10 +7096,13 @@ public List<SpellAbility> getAllPossibleAbilities(final Player player, final boo
             }
         }
 
-        if (isInPlay() && isFaceDown() && isManifested()) {
-            ManaCost cost = oState.getManaCost();
-            if (oState.getType().isCreature()) {
-                abilities.add(CardFactoryUtil.abilityManifestFaceUp(this, cost));
+        if (isInPlay() && isFaceDown() && oState.getType().isCreature())
```

Comment:
Imo it also needs a check for manacost here
e. g. currently I can unmanifest _Asmoranomardicadaistinaculdacar_ for {0}

> If the card representing that permanent isn't a creature card or it doesn't have a mana cost, it can't be turned face up this way.